baseplate/frameworks/queue_consumer/kafka.py

```diff
@@ -235,17 +300,22 @@ def new(
         :param kafka_consume_batch_size: The number of messages the `KafkaConsumerWorker`
             reads from Kafka in each batch. Defaults to 1.
         :param message_unpack_fn: A function that takes one argument, the `bytes` message body
-            and returns the message in the format the handler expects. Defaults to `json.loads`.
+            and returns the message in the format the handler expects. If no input is provided,
+            a function is automatically picked one based on whether a schema_registry was provided or not.
```

Comment:
If no value is provided, we will use avro decode_message when using schema registry, and json.loads otherwise

?